contracts/libraries/Ticks.sol

```diff
@@ -203,8 +206,8 @@ library Ticks {
             IRangePoolStructs.PoolState memory,
             IRangePoolStructs.SwapCache memory
     ) {
-        if (zeroForOne ? priceLimit >= pool.price || pool.price == TickMath.MIN_SQRT_RATIO
-                       : priceLimit <= pool.price || pool.price == TickMath.MAX_SQRT_RATIO)
+        if (zeroForOne ? priceLimit >= pool.price
+                       : priceLimit <= pool.price)
         {
```

Comment:
The latter validation is no longer necessary as the pool price will never reach the min and max prices defined in `TickMath` unless 887272 % tickSpacing == 0.